mod/wet4/views/default/page/components/module.php

```diff
@@ -143,6 +143,17 @@
         $header = elgg_format_element('h2', ['class' => 'panel-title'], $title);
     }
 
+    // CL 23082022 - component determines if the h2 title is Colleagues or collegues. Appends the pager owner 
+    $owner = elgg_get_page_owner_entity()->username;
```

Comment:
I can see the header display in French in my environment. 
![image](https://user-images.githubusercontent.com/7603237/186241364-c2e57ef1-5caa-4131-8f67-79b254f71e4c.png)
I'm not sure why it won't display for Shea. I'll keep looking into this.

For review, I would update `username` here to just `name`. This should display the user's preferred display name set in their profile. Usernames on the platform are system generated so I think displaying `name` here would be best.

Reply:
Swapped username to the name prop
![image](https://user-images.githubusercontent.com/17506947/186248625-4f979436-791e-473c-9f4b-eab2d26318c3.png)
![image](https://user-images.githubusercontent.com/17506947/186248677-28894209-b9e5-49a6-a965-4b3ec075b0d7.png)